modules/registry-checking-the-status-of-registry-pods.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * registry/accessing-the-registry.adoc
+
+[id="checking-the-status-of-registry-pods_{context}"]
+= Checking the status of the registry Pods
+
+As a cluster administrator you can list the image registry Pods running in the `openshift-image-registry` project and check their status.
```

Comment:
As a cluster administrator, you can

Reply:
Thank you for your review! I have now resolved this suggestion.